nimbus-core/src/main/java/com/antheminc/oss/nimbus/channel/web/WebActionController.java

```diff
@@ -190,10 +211,40 @@ else if(o.getValue() instanceof Param) {
 		return ResponseEntity.status(HttpStatus.OK).body(null);
 	}
 	
+	@RequestMapping(value=URI_PATTERN_P+"/event/upload",consumes = MediaType.MULTIPART_FORM_DATA_VALUE,  produces="application/json", method=RequestMethod.POST)
+	public Object handleUpload(HttpServletRequest req, @RequestParam("pfu") MultipartFile file, @RequestParam("domain") String domain) {
+		if (file.isEmpty()) {
+			return new Holder<>(false);
+		}
+		
+		Command command = this.builder.build(req);
+		Command uploadCommand = CommandBuilder.withPlatformRelativePath(command, Type.PlatformMarker, "/" + domain).getCommand();
+		uploadCommand.setRequestParams(command.getRequestParams());
+		uploadCommand.setAction(Action._new);
+		
+		try {
+			String ext = FilenameUtils.getExtension(file.getName());
+			Importer importer = getFileImporter(ext);
+			if (null == importer) {
+				throw new UnsupportedOperationException("Upload for file types of \"" + ext +"\" is not supported.");
```

Comment:
Please use f/w exceptions. This could be InvalidConfigException

Reply:
Fixed.